File: PHP/classExercicio2.php
"""
    Classe: Conta
    Atributos:
    - numero : int
    - banco : string
    - saldo : double
    Métodos:
    + depositar(valor:double):void #Printe o saldo
    + sacar(valor:double):void #Printe o saldo
    """
<?php
    class Conta {
        private $numero = 0;
        private $banco  = 'Numbank';
        private $saldo  = 0.0;
        function depositar ($valor){
            $this->saldo +=$valor;
            echo 'R$'.$this->saldo;
        }
        function sacar($valor){
            $this->saldo -= $valor;
            echo 'R$'.$this->saldo;
        }
    }
    $conta1 = new conta();
    $conta1 ->depositar(100);
    $conta1 ->sacar(50);

File: Python/classExercicio2.py
"""
Classe: Conta
Atributos:
- numero : int
- banco : string
- saldo : double
Métodos:
+ depositar(valor:double):void #Printe o saldo
+ sacar(valor:double):void #Printe o saldo
"""
class Conta :
    def __init__(self):
        self.__numero = 0
        self.__banco  = ''
        self.__saldo  = 0
    def depositar (self,valor):
        self.__saldo += valor
        print(self.__saldo)
    def sacar (self,valor):
        self.__saldo -= valor
        print(self.__saldo)
if __name__=='__main__':
    conta1 = Conta()
    conta1.depositar(100)
    conta1.sacar(5)